docs/references/vod/patch.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: PATCH updating storage
+---
+
+
+`PATCH /asset/:id -d {"storage": {"ipfs": {}}` can be used to modify any `asset` that was stored with On Demand. It is a single endpoint that can update only the following mutable properties of an asset:
+
+- `name`
+- `storage`
+- `meta`
+
+
+
+## Update Stored Asset
+
+> **NOTE:** When updating the asset information, the `storage` object must be included in the body of the request
+
+## Updating Asset Name
+
+This will allow the name of the stored asset in Livepeer Studio to be changed.
+
+### Request
+
+```bash
+curl -X PATCH https://livepeer.studio/api/asset/{id} \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer {api-key}' \
+-d {
+  "name": "New name"
+  "storage": {"ipfs": {}}
+  }
+```
+
+## Updating Storage
```

Comment:
We need instructions on how to track the progress of the storage update. Maybe we should have a separate guide for it tho, since it's less of an API reference and more of a "how to" guide.